modules/raycast/SCsub

```diff
@@ -86,6 +86,20 @@ if env["builtin_embree"]:
         # Embree needs those, it will automatically use SSE2NEON in ARM
         env_thirdparty.Append(CPPDEFINES=["__SSE2__", "__SSE__"])
 
+    if not env.msvc:
+        env_thirdparty.Append(
```

Comment:
Maybe add a comment to document why those are here? (mentioning that it's synced with the upstream buildsystem is enough)